### PR TITLE
Update explainer to reflect API implemented in Chromium

### PR DESCRIPTION
**Pull request by guidou**
Update to the explainer using the API we are implementing in Chromium.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -10,104 +10,209 @@ We need an API for processing media that:
...
+    let senderTransform = new TransformStream({
+      start() {
+        // Called on startup.
+      },
+
+      async transform(chunk, controller) {
+        let view = new DataView(chunk.data);
+        // Create a new buffer with 4 additional bytes.
+        let newData = new ArrayBuffer(chunk.data.byteLength + 4);
+        Let newView = new DataView(newData);
```
Comment (leader22) [inline explainer.md]:
```suggestion
        let newView = new DataView(newData);
```

File: explainer.md [hunk 2]
```diff
@@ -10,104 +10,209 @@ We need an API for processing media that:
...
+
+        // Fill the new buffer with a negated version of all
+        // the bits in the original frame.
+        for (let i = 0; i < chunk.data.byteLength; ++i)
+          newView.setInt8(i, ~view.getInt8(i));
+        // Set the padding bytes to zero.
+        For (let i = 0; i < 4; ++i)
```
Comment (leader22) [inline explainer.md]:
```suggestion
        for (let i = 0; i < 4; ++i)
```

File: explainer.md [hunk 3]
```diff
@@ -10,104 +10,209 @@ We need an API for processing media that:
...
+let [track] = stream.getTracks();
+let videoSender = pc.addTrack(track, stream)
+let senderStreams = videoSender.getEncodedVideoStreams();
+
+// Do ICE and offer/answer exchange.
+
+senderStreams.readable
+  .pipeThrough(senderTransform)
+  .pipeTo(senderStreams.writable);
 }
```
Comment (leader22) [inline explainer.md]:
```suggestion

```

File: explainer.md [hunk 4]
```diff
@@ -10,104 +10,209 @@ We need an API for processing media that:
...
+      controller.enqueue(chunk);
+      },
+    });
+
+    let receiver_streams = video_receiver.createEncodedVideoStreams();
+    receiver_streams.readable
+      .pipeThrough(my_transform)
+      .pipeTo(receiver_streams.writable);
+  }
+}
```
Comment (leader22) [inline explainer.md]:
```suggestion
    },
  });

  let receiver_streams = video_receiver.createEncodedVideoStreams();
  receiver_streams.readable
    .pipeThrough(my_transform)
    .pipeTo(receiver_streams.writable);
};
```

File: explainer.md (hunk 1 above)
Comment (guidou) [reply explainer.md]: done

**Comment by guidou** [comment]
thanks for the review, @leader22

**Comment by alvestrand** [comment]
I'll put the factory method back in as a new section.

**Pull request merged**